src/Configuration.php

```diff
@@ -24,6 +28,15 @@ class Configuration
     /**
      * The cache driver implementation that is used for query result caching.
      *
+     * @var CacheItemPoolInterface|null
+     */
+    protected $resultCache;
```

Comment:
```suggestion
    private $resultCache;
```

Reply:
I tried mimic the style of this class here, but it probably makes sense to make this property private.

Reply:
Moreover, `$imiddlewares` is already private :)